11월 10일 창호 index.css

Algorgasm opened the pull request.

Algorgasm committed:
11월 10일 창호 index.css

heejeongMin commented on css/index.css:
```diff
@@ -24,11 +24,8 @@ figure figcaption{
-	0% {transform: scale(0.7);}
-	100% {transform: scale(1.2);}
-}
+
```
내 브랜치에 있던 css파일이 이전 버전이여서 빠진 라인은 수정했어요

heejeongMin committed:
11_10 창호 버전 추가 수정했습니다.

수정 1.  내 브랜치에 있던 css파일이 이전 버전이여서 라인 및 제거해야할 속성은 수정해 놓았습니다. (ex. 라인 27)
수정 2.   figure.fig2 img 에 object-fit: cover 속성은 제거 했습니다. 라인 20번에 default로 이미 정의 되어 있습니다.
수정 3.  figure.fig2::before 의 브라우저호환성때문에 넣어놓은 webkit prefix는 일단은 삭제했습니다. 나중에 cdn 방식으로 한꺼번에 넣으려구요.

heejeongMin committed:
Merge pull request #18 from heejeongMin/heejeongMin-patch-1

11_10 창호 버전 추가 수정했습니다.
